site/content/en/docs/Installation/Install Agones/helm.md

```diff
@@ -35,24 +38,68 @@ pool for Agones, see the [Agones installation instructions]({{< relref "../_inde
 The command deploys Agones on the Kubernetes cluster with the default configuration. The [configuration](#configuration) section lists the parameters that can be configured during installation.
 
 {{< alert title="Tip" color="info">}}
-List all releases using `helm list`
+List all releases using `helm list --all-namespaces`
 {{< /alert >}}
 
+### Namespaces
+
+By default Agones is configured to work with game servers deployed in the `default` namespace. If you are planning to use another namespace you can configure Agones via the parameter `gameservers.namespaces`.
+
+For example to use `default` **and** `xbox` namespaces:
+
+```bash
+$ kubectl create namespace xbox
+$ helm install my-release agones/agones --set "gameservers.namespaces={default,xbox}" --namespace agones-system
+```
+
 {{< alert title="Note" color="info">}}
-If you are installing a development build of Agones (i.e. not the {{< release-version >}} release), you will need to install Agones the following way:
+You need to create your namespaces before installing Agones.
 {{< /alert >}}
 
+If you want to add a new namespace afterward simply upgrade your release:
+
 ```bash
-$ cd install/helm/
-$ helm install --name my-release --namespace agones-system agones --set agones.image.tag={{< release-version >}}-481970d
+$ kubectl create namespace ps4
+$ helm upgrade my-release agones/agones --set "gameservers.namespaces={default,xbox,ps4}"
```

Comment:
While we're at it, we might want to add `--reuse-values` to this command-line, so that people using it don't accidentally remove other settings that have been applied to the Helm chart.

Reply:
Ooh, that's an excellent suggestion. Making this adjustment now!